hpedockerplugin/volume_manager.py

```diff
@@ -1791,7 +1791,7 @@ def _find_rcg(self, rcg_name):
         return rcg_info
 
     # TODO: Need RCG lock in different namespace. To be done later
```

Comment:
since the different locking namespace has been used already.. you can remove the TODO here ::-)